server/channels/web/webhook.go

```diff
@@ -26,7 +26,15 @@ func incomingWebhook(c *Context, w http.ResponseWriter, r *http.Request) {
 	params := mux.Vars(r)
 	id := params["id"]
 
-	r.ParseForm()
+	err := r.ParseForm()
```

Comment:
`err` also gets used as a variable name in L39. You can either rename that one to `appErr` or change this one to `nErr`.

Reply:
Sure will rename the variable.